contracts/FlowIDTableStaking.cdc

```diff
@@ -878,13 +841,35 @@ pub contract FlowIDTableStaking {
 
             emit NewDelegatorCutPercentage(newCutPercentage: FlowIDTableStaking.nodeDelegatingRewardCut)
         }
+
+        pub fun setClaimed() {
+
+            let claimedNetAddressDictionary = FlowIDTableStaking.account.load<{String: Bool}>(from: /storage/networkingAddressesClaimed)
+                ?? panic("Invalid path for networking address dictionary")
+
+            let claimedNetKeyDictionary = FlowIDTableStaking.account.load<{String: Bool}>(from: /storage/networkingKeysClaimed)
+                ?? panic("Invalid path for networking key dictionary")
+
+            let claimedStakingKeysDictionary = FlowIDTableStaking.account.load<{String: Bool}>(from: /storage/stakingKeysClaimed)
+                ?? panic("Invalid path for staking key dictionary")
+
+            for nodeID in FlowIDTableStaking.nodes.keys {
+                claimedNetAddressDictionary[FlowIDTableStaking.nodes[nodeID]?.networkingAddress!]
+                claimedNetKeyDictionary[FlowIDTableStaking.nodes[nodeID]?.networkingKey!]
+                claimedStakingKeysDictionary[FlowIDTableStaking.nodes[nodeID]?.stakingKey!]
+            }
+
+            FlowIDTableStaking.account.save(claimedNetAddressDictionary, to: /storage/networkingAddressesClaimed)
+            FlowIDTableStaking.account.save(claimedNetKeyDictionary, to: /storage/networkingKeysClaimed)
+            FlowIDTableStaking.account.save(claimedStakingKeysDictionary, to: /storage/stakingKeysClaimed)
```

Comment:
Maybe use `updateClaimed` here?

Reply:
I thought about using that, but this seemed like a more efficient way to do it, though I guess we'll only run it once so it might not be an issue